### ISSUE TITLE
Open resource search string abbreviation default, ignore case.

### ISSUE THREAD

**Issue by alagatar** [SlavaRa/fd-quicknavigate-plugin#35]
I long use Sublime text 3, and their ctrl+p is awesome speed up work process. For FD I found your plugin. This awesome, but can be settings for search string was abbreviation default?
Thanks

**Comment by SlavaRa** [comment]
UPPERCASE for search by abbreviation

**Comment by alagatar** [comment]
yes, i know this. Can you add option plugin setting for default abbreviation?

**Comment by SlavaRa** [comment]
You are russion?

**Comment by alagatar** [comment]
Да. В идеале было бы, добавить настройку в плагин, которая делает по умолчанию все что ты вводиш в поле как абревиатуру. Я работал с sublime text, и там просто идеальный вариант быстрого поиска, перемещения, ctrl+p набрал пару символов и все перешел к файлу.

**Comment by SlavaRa** [comment]
а как должен вести себя поиск при вводе, например, dobc, должен ли от показывать DIsplayObjectContainer или только DisplayObjectBaseController, будет считать, что у нас только два таких класса?

**Comment by alagatar** [comment]
Оба показываются

**Comment by SlavaRa** [comment]
Вот именно из-за такого поведения, оно не однозначное, я убирал эту опцию. 
Но хорошо, я подумаю сегодня-завтра и отпишу сюда к чему пришел.

**Comment by alagatar** [comment]
Спасибо. Возможно поможет, посмотри как себя ведет поиск в Sublime text

**Comment by SlavaRa** [comment]
А может быть есть способ снять видео или gif сделать?

**Comment by alagatar** [comment]
Ок, сейчас сделаю

**Comment by alagatar** [comment]
Записал небольшое видео, вот на нем и видно почему хочеться перейти на flashdevelop, лаги.
https://db.tt/2xJrmD7F

**Comment by alagatar** [comment]
Имеется ли прогресс по даному вопросу и как сложно будет реализовать данный функционал.

**Comment by SlavaRa** [comment]
Релиз с этим функционалом будет в понедельник, извини, забыл написать)

**Comment by alagatar** [comment]
Это просто замечательная новость! Я правельно понял, что в понедельник будет релиз flashdevelop(не плагина) где это вшито?

**Comment by SlavaRa** [comment]
не, будет релиз плагина с этим функционалом, про полный перенос всего в FD пока обсуждается

**Comment by alagatar** [comment]
А можно ли будет отключить встроеный в FD Open resource (ctrl+r) и потом заменить плагином?

**Comment by SlavaRa** [comment]
QuickNavigate немного дополнит нативный функционал незаметно от пользователя,.. если такое поведение всех устроит, то я перенесу в редактор, вот так решим)

**Comment by alagatar** [comment]
Механизм который я хотел бы чтобы появился в плагине имеет общее название fuzzy search.
https://www.youtube.com/watch?v=eOusbX6L8CI
Есть много реализаций этого алгоритма, но к сожалению мои знания c# не позволяют чтобы самому сделать подобный плагин.

**Comment by SlavaRa** [comment]
привет, что-то у меня не задавалось со временем, я постараюсь сегодня-завтра посмотреть... извини, что долго(

**Comment by SlavaRa** [comment]
Это действительно интересно, я постараюсь выкроить время завтра вечером.

**Comment by SlavaRa** [comment]
> А можно ли будет отключить встроеный в FD Open resource (ctrl+r) и потом заменить плагином?
это но я принял решение отказаться, т.к. в редакторе есть нативный, и не хочется поддерживать два

> QuickNavigate немного дополнит нативный функционал незаметно от пользователя
так сделать не вышло, вернулись в пункту один(

> Механизм который я хотел бы чтобы появился в плагине имеет общее название fuzzy search.
я почти закончил с кодом, остались тесты, поэтому если вы не против, то я закрою эту задачу и создам отдельную на внедрение "fuzzy search", хорошо?

по поводу Open Resource, я покажу ребятам решение https://github.com/fdorg/flashdevelop/issues/913, а там как повезет))
